VibeCodeApp/VibeCodeApp.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net9.0-android</TargetFrameworks>
```

Comment:
Make sure to keep ios and mac catalyst, but add a condition to run on windows and macos only. See what we did for the windows condition.